src/ChannelServer/Util/GmCommands.cs

```diff
@@ -49,7 +49,7 @@ public GmCommands()
 			Add("speed", "<speed>", HandleSpeed);
 			Add("iteminfo", "<name>", HandleItemInfo);
 			Add("monsterinfo", "<name>", HandleMonsterInfo);
-			Add("go", "<destination>", HandleGo);
+			Add("tele", "<destination>", HandleTele);
```

Comment:
@Cosrnos Aura has a go command as well. Warp takes map and coordinates, while go warps to pre-defined destinations. E.g. `>go dun`/`/go c_Klaipe`